docs/additional-resources/blue-green-migration.md

```diff
@@ -4,24 +4,62 @@
 
-Additionally, for deployments that require change in the data schema or broad stroke changes to the database in order for the application to function, we build an empty copy of the data stores (DynamoDB and Elasticsearch). Then we perform a migration of information from the old data store into the newly created one. This migration passes all of the data through a lambda to update and verify the data before saving it into a new data store.
+Additionally, for deployments that require change in the data schema or elasticsearch mappings in order for the application to function, we build an empty copy of the data stores (DynamoDB and Elasticsearch). Then we perform a migration of information from currently running DynamoDB database table (*source table*) into the newly created, empty one (*destination table*). This migration passes all of the data through a lambda to update and verify the data before saving it into the destination table.
 
 ## Automated Migration Steps
 
 **All of the automated migration steps are now handled by CircleCI, as defined in `config.yml`.**
 
 Some key steps, excluding automated tests, include:
 
+### Manual Steps required before attempting an automated Migration
+
+If the destination table exists in either `us-east-1` or `us-west=1`, you will have to delete it before kicking off a CircleCI deployment.
+
+1. Identify the **destination table**. For example, if we're performing a deployment to `dev`:
+
+    ```bash
+    $ ./scripts/dynamo/get-source-table.sh dev
+    efcms-dev-beta
+    $ ./scripts/dynamo/get-destination-table.sh dev
+    efcms-dev-beta
+    ```
+
+    Both should be the same value, and that is what the application is currently using as it its main data store. If they are both `alpha`, then the destination table is `beta`. If they are both `beta`, the destination table is `alpha`.
+
+    If they are different, inspect the Lamdba's environment config to determine what table the application is currently using. If that is `beta`, then the destination table is `alpha`, and vice versa.
+
+2. Check to see if the **destination table** exists. You can do this via the AWS Console or AWS CLI:
+
+    ```bash
+    $ aws dynamodb describe-table --table-name efcms-dev-alpha
+    ```
+
+    If you get an error, `An error occurred (ResourceNotFoundException) when calling the DescribeTable operation: Requested resource not found: Table: efcms-dev-alpha not found`, then you are ready to proceed with the deployment.
+
+    If you do not, then it exists, and you must delete the table.
+
+3. If it exists, delete the **destination table** in both `east` and `west`. You can do this via the AWS Console or AWS CLI:
+
+    ```bash
+    $ aws dynamodb delete-table --table-name efcms-dev-alpha --region us-west-1
+    $ aws dynamodb delete-table --table-name efcms-dev-alpha --region us-east-1
+    ```
+
+    NOTE: after deleting the `west` table, you may have to wait a few minutes before you can delete the `east` table.
```

Comment:
We also need to delete the destination elasticsearch domain

Reply:
I don't think this is true anymore - I think it won't exist because it's not being created if it's not needed.